Automattic/wp-calypso
# Analytics Fix: Bring hosting decider to signup framework

Related to #

## Proposed Changes

* Moves the hosting decider step to the signup framework
* Cleanup all previous code related to the hosting decider change
* Routes to relevant stepper flow based on button click

## Testing Instructions
- Go to `/start/hosting-start`
- Signup
- Make sure the hosting-decider step visible
- Click on CTAs and make sure that you are routed to the route path.

Pending: Change landpack CTA urls and also include the toStepper parameter


## Pre-merge Checklist
